### DISCUSSION TITLE
How Do You Handle Missing or Corrupted Data in a Dataset?

### DISCUSSION THREAD

**Discussion by luffy-Monkey123** [luffy-Monkey123/python-cool-programs#16] [Q&A] [answered]
How Do You Handle Missing or Corrupted Data in a Dataset?

**Comment by RUTAGENGWA-ASANTE-BRUCE** [comment, marked as answer]
One of the easiest ways to handle missing or corrupted data is to drop those rows or columns or replace them entirely with some other value.

There are two useful methods in Pandas:

1. IsNull() and dropna() will help to find the columns/rows with missing data and drop them
2. Fillna() will replace the wrong values with a placeholder value
